theme/css/chrome.css

```diff
@@ -390,6 +390,12 @@ ul#searchresults span.teaser em {
     display: flex;
     color: var(--sidebar-non-existant);
 }
+
+.chapter li :hover {
+    color: var(--sidebar-active);
+    cursor: pointer;
```

Comment:
Non-blocking, but it might be worth adding `user-select: none;` to this or the div/non-hover version of it. Because you cannot select the links, but the expandable headers you can click and drag to highlight.

Reply:
Good point , just did